lib/trento/application/integration/discovery/policies/host_policy.ex

```diff
@@ -56,7 +60,12 @@ defmodule Trento.Integration.Discovery.HostPolicy do
         "payload" => payload
       }) do
     subscriptions =
-      Enum.map(payload, fn subscription -> parse_subscription_data(agent_id, subscription) end)
+      payload
+      |> Enum.map(fn subscription -> SubscriptionDiscoveryPayload.new(subscription) end)
+      |> Enum.reject(fn {tag, _} -> tag != :ok end)
+      |> Enum.map(fn {:ok, subscription} ->
+        subscription |> Map.from_struct() |> Map.put(:host_id, agent_id)
```

Comment:
I know that this is inherited, but I wonder if we really need to append this `host_id` value to the SlesSubscriptions value object. We already have the `host_id` in the host aggregate, so it looks redundant. 

Anyway, nevermind. We don't need to really touch it as it works fine, and it is a small detail hehe